cmd/limactl/editflags/editflags.go

```diff
@@ -75,6 +77,15 @@ func RegisterEdit(cmd *cobra.Command, commentPrefix string) {
 	_ = cmd.RegisterFlagCompletionFunc("disk", func(*cobra.Command, []string, string) ([]string, cobra.ShellCompDirective) {
 		return []string{"10", "30", "50", "100", "200"}, cobra.ShellCompDirectiveNoFileComp
 	})
+
+	flags.String("vm-type", "", commentPrefix+"Virtual machine type")
```

Comment:
`--vm-type` is not guaranteed to work as an edit flag.

e.g., this will probably not work
```
limactl create --vm-type qemu
limactl edit --vm-type vz
```

Probably we need a warning when `--vm-type` is specified as an edit flag for an existing instance, but it can be worked out later.